src/pages/Cart/Cart.css

```diff
@@ -1,6 +1,7 @@
 .cart-wrapper {
   width: 90%;
   margin: 5rem auto 7rem auto;
+  height: 100vh;
```

Comment:
use `min-height` if this is a page wrapper. other wise content will overflow if someone adds too many things

Reply:
Done